examples/seq2seq/utils.py

```diff
@@ -42,11 +42,11 @@ def encode_file(
     examples = []
     for text in tqdm(lns, desc=f"Tokenizing {data_path.name}"):
         tokenized = tokenizer(
-            [text],
+            [" " + text],
             max_length=max_length,
             padding="max_length" if pad_to_max_length else None,
             truncation=True,
-            add_prefix_space=True,
+            # add_prefix_space=True,
```

Comment:
Why this change?

Reply:
It now creates a warning, and I lazily haven't checked whether the outputs have changed.
I will resolve in this PR.

Reply:
I can't reproduce the warning. going back to add_prefix_space=True.